app/(game)/Game.tsx

```diff
@@ -219,63 +159,52 @@ export default function Game() {
 
   return (
     <div className="grid h-screen place-items-center px-5 py-5">
-      {ready ? null : <div className="animation-wrapper" />}
-      <div className="game-wrapper">
-        {gameMode === "nemein" ? null : (
-          <div className="top">
-            <div className="text-slate-100 text-xl">
-              <p className="font-bold">LEVEL</p>
-              <p className="font-medium">{gameData.level}</p>
-            </div>
-            <div className="text-slate-100 text-xl">
-              <p className="font-bold">SCORE</p>
-              <p className="font-medium">{gameData.score}</p>
-            </div>
-          </div>
-        )}
-        <div className="held">{gameData.held}</div>
-        <div className="main">
-          {ready && !active ? (
-            <ControlPrompt isOver={isOver.current} restartGame={restartGame} />
-          ) : null}
-          {gameData.main}
-        </div>
-        <div className="queue">{gameData.queue}</div>
-      </div>
       {ready ? null : (
-        <button
-          className={"start-button button button-light"}
-          onClick={() => {
-            const startTimeline = anime.timeline({
-              easing: "easeInOutCubic",
-              duration: 750,
-            });
-
-            startTimeline
-              .add({
-                targets: ".start-button",
-                opacity: 0,
-                zIndex: 0,
-              })
-              .add({
-                targets: ".animation-wrapper",
-                height: 480,
-                width: 240,
-              })
-              .add({
-                targets: ".game-wrapper",
-                opacity: 1,
-                zIndex: 40,
-                complete: startGame,
+        <>
+          <div className="animation-wrapper" />
+          <progress className="init-progress" value="10" max="100" />
+          <button
+            className={"start-button button button-light"}
+            onClick={() => {
+              const startTimeline = anime.timeline({
+                easing: "easeInOutCubic",
+                duration: 500,
```

Comment:
Same as above

Reply:
I also noticed that we're using quite a lot of magic numbers here. It might not worth creating consts for all of them though let's do so for the ones that you think might get changed frequently

Reply:
> I also noticed that we're using quite a lot of magic numbers here. It might not worth creating consts for all of them though let's do so for the ones that you think might get changed frequently

I wasn't too sure about these numbers because they are CSS values. I think having consts for the duration of the animations make the most sense. I can add some comments for the other cases.